core/server/data/schema/schema.js

```diff
@@ -351,6 +351,9 @@ module.exports = {
         subscription_id: {type: 'string', maxlength: 255, nullable: false, unique: false},
         plan_id: {type: 'string', maxlength: 255, nullable: false, unique: false},
         status: {type: 'string', maxlength: 50, nullable: false},
+        cancel_at: {type: 'dateTime', nullable: true},
```

Comment:
Out of interest, what are the `cancel_at` and the `canceled_at` fields going to be used for?

Reply:
`cancel_at` can be used as a more "sure" substitute instead of `current_period_end`. The other one `canceled_at` - I'm still to ask @peterzimon if we would possibly want to show something to a member that canceled in the past. So, that info would be useful to show (when they canceled)

Reply:
Yeah I think `canceled_at` might be useful in the future. But for me `current_period_end` is much more understandable than `cancel_at`

Reply:
Ok. In that case, I'd keep the change to the minimum and only add what we absolutely need for this feature - `cancel_at_period_end` flag. The rest can come in the future if we find concrete use-cases for them.